packages/shell-api/src/shell-instance-state.ts

```diff
@@ -362,6 +365,7 @@ export default class ShellInstanceState {
       });
     }
 
+    Object.assign(contextObject, { log: this.shellLog });
```

Comment:
This is an involved way of writing

```suggestion
    contextObject.log = this.shellLog;
```

🙂